wrappers/wrapper.rb

```diff
@@ -64,19 +62,6 @@ def assert_vehicles_no_alternative_skills(vrp)
       vrp.vehicles.none?{ |vehicle| vehicle.skills.size > 1 }
     end
 
-    def assert_no_direct_shipments(vrp)
-      vrp.shipments.none?{ |shipment| shipment.direct }
-    end
-
-    def assert_no_pickup_timewindows_after_delivery_timewindows(vrp)
```

Comment:
I think this should still be checked.

Reply:
This assert has been moved to validate_data.